cmd/podman/runlabel.go

```diff
@@ -97,6 +98,12 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 	}
 	defer runtime.DeferredShutdown(false)
 
+	if c.Authfile != "" {
+		if _, err := os.Stat(c.Authfile); err != nil {
+			return errors.Wrapf(err, "error getting authfile %s", c.Authfile)
+		}
+	}
+
```

Comment:
We seem to be doing this in many of the commands.  Should we make a little utility function in cmd/podman/utils.go?

Reply:
I think I should add this kind of function in buildah and vendor from there. And add check nonexist authfile in buildah, too.

Reply:
That's fine by me if that works better.